### PR TITLE
My hw 03

### PR DESCRIPTION
**Pull request by TimeForward**

### CODE REVIEW

**Comment by vladhilko** [comment]
You have conflicts. You need to do rebase or merge current main branch to your branch

**Comment by vladhilko** [comment]
Or remove unrelated files from your PR 

Screenshot 2022-03-14 at 10 28 35

**Comment by vladhilko** [comment]
CI is red

**Comment by TimeForward** [comment]
fixed.
appreciate the understanding

**Comment by TimeForward** [comment]
final attempt

**Comment by vladhilko** [comment]
Are you going to fix CI? You just need to run `rubocop -a`

**Comment by TimeForward** [comment]
Thank you for your email.
Actually that was the one of the reasons  i failed.
I believe , I am excluded from the course now.

I have been running rubocop -a and rubocop -A locally, and it showed all
fixed, zero conflicts, but when I do PRs - runs fail

чт, 17 мар. 2022 г., 22:08 Vlad Hilko ***@***.***>:

File: HW03/roman_nasyrov/roman_nasyrov_hw_03_t_02.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# declaring class
+class Homework3
+  REGEX = %r{^([\d+.\d+]+).*(?<=\[)(.*)(?=\]).*(?<=\s)((?:/\w+)*?)(?=\s).*$}.freeze
```
Comment (vladhilko) [inline HW03/roman_nasyrov/roman_nasyrov_hw_03_t_02.rb:5]: There's only one rubocop error, maybe you didn't save the changes? 🤔 
```suggestion
  REGEX = %r{^([\d+.\d+]+).*(?<=\[)(.*)(?=\]).*(?<=\s)((?:/\w+)*?)(?=\s).*$}
```

**Comment by B0RlS** [comment]
@TimeForward HW Failed

**Pull request closed without merging**